src/backgroundScripts/background.js

```diff
@@ -74,3 +81,26 @@ tabs.onActivated.addListener(async (activeInfo) => {
  * Open the sidebar when the button gets clicked
  */
 button.onClicked.addListener(() => browser.sidebarAction.open());
+
+/*
+ * Message handler
+ */
+browser.runtime.onMessage.addListener(async (msg) => {
+	if (msg.aim === 'getResource') {
+		return model.getResource(msg.uri, msg.reload);
+	}
+	if (msg.aim === 'isWhitelisted') {
+		return model.isWhitelisted(msg.uri);
+	}
+	if (msg.aim === 'createHighlight') {
+		return model.createHighlight(msg.uri,msg.viewpoint,msg.topic,msg.coordinates);
+	}
+	if (msg.aim === 'removeHighlight') {
+		return model.removeHighlight(msg.uri,msg.viewpoint,msg.topic,msg.fragId);
+	}
+	if (msg.aim === 'setHLNumber') {
+		if (msg.count && msg.tabId) {
+			return setHLNumber(msg.count,msg.tabId);
+		}
+	}
```

Comment:
Why not a switch/case structure?

Reply:
You are right, it should be. This is a mere move of a part of code in another file, which extended an [existing](https://github.com/Hypertopic/LaSuli/pull/61/commits/7c8dc841b371f4bde66a8d14933e55e998b69adf#diff-2b6e46436cc8167726516dc3bacdf6acL247) `if` series, and I didn't transform it in a switch/case block.
I'll commit it, not before tomorrow.

Reply:
Done in b1d41be